jobs/kola-openstack.Jenkinsfile

```
@@ -43,97 +41,100 @@ properties([
     durabilityHint('PERFORMANCE_OPTIMIZED')
 ])
 
-currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
+lock(resource: "build-${params.STREAM}-${params.ARCH}") {
```

Let's adjust the lock name slightly.

- Switch to denote which job we are locking (`kola-openstack`).
- We want a single run on each architecture at any given time. Then the code will switch the region based on the architecture.

```suggestion
lock(resource: "kola-openstack-${params.ARCH}") {
```